src/Jobs/ImportJob.php

```
@@ -63,6 +64,17 @@ public function handle()
                 ImportExport::COLUMN_STATUS => ExportStatus::Successful,
                 ImportExport::COLUMN_COMPLETED_AT => now(),
             ]);
+
+            if ($importObject->failures()->isNotEmpty()) {
+                $this->export->update([
```

You can throw a Exception with message: 
```
$importObject
                        ->failures()
                        ->map(fn ($item, $key) => $item->errors())
                        ->flatten()
                        ->implode(', ')
```